ee_tests/src/page_objects/space_settings.page.ts

```diff
@@ -0,0 +1,53 @@
+import { browser, element, by, By, ExpectedConditions as until, $, $$, ElementFinder } from 'protractor';
+import * as support from '../support';
+import { TextInput, Button, BaseElement, BaseElementArray, Clickable } from '../ui';
+import { BasePage } from '.';
+
+
+export class SpaceSettings extends BasePage {
+  settings = new BaseElement($('.pficon-settings'), 'Settings');
+  areasTab = new Button (element(by.xpath('//span[contains(text(),\'Areas\')]')), 'Areas');
```

Comment:
Please avoid xpath wherever possible, use `by.cssContainingText()` instead

Reply:
@ppitonak it is for Space settings (adding areas and Collaborators to space) and then added areas and collaborators can be assigned to a workitem.